templates/chrony.conf.epp

```diff
@@ -120,3 +128,12 @@ smoothtime <%= $chrony::smoothtime %>
 # https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html#rtconutc
 rtconutc
 <% } -%>
+<% if ! $chrony::hwtimestamps.empty { -%>
```

Comment:
Would `unless` be more readable than `if !`?

Reply:
Interesting that you should mention that. I myself prefer unless before if ! in any language that supports it. But, I know people that hates unless. :)

Reply:
There are more 'if !' in the template, I can add a separate PR later that changes them all to unless, and se if someone else objects.